ppl/zqd/db/filedb/migrate.go

```diff
@@ -140,3 +127,38 @@ func migrateV5(data []byte) (int, []byte, error) {
 	data, err := json.Marshal(db)
 	return 5, data, err
 }
+
+type dbdataV5 struct {
+	Version   int     `json:"version"`
+	SpaceRows []rowV5 `json:"space_rows"`
+}
+
+type rowV5 struct {
+	ID       api.SpaceID       `json:"id"`
+	DataURI  iosrc.URI         `json:"data_uri"`
+	Name     string            `json:"name"`
+	ParentID api.SpaceID       `json:"parent_id"`
+	Storage  api.StorageConfig `json:"storage"`
+	TenantID auth.TenantID     `json:"tenant_id"`
+}
+
+func migrateV6(data []byte) (int, []byte, error) {
+	var dbv5 dbdataV5
+	if err := json.Unmarshal(data, &dbv5); err != nil {
+		return 0, nil, err
+	}
+	var db dbdata
+	db.Version = 6
+	db.SpaceRows = make([]schema.SpaceRow, 0, len(dbv5.SpaceRows))
+	for _, r := range dbv5.SpaceRows {
```

Comment:
This probably doesn't matter since I doubt there are any zqd instances with a subspace in the wild... but should we drop subspaces here?

Reply:
I convinced myself that it was ok. We never exposed it in Brim, except for some internal demos. Plus, the subspace was essentially just a filtered view to the parent space's data - so removing the subspace doesn't actually delete any data.